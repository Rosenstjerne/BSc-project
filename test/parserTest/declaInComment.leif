#* hej # hej *# var int a; # var int b;


a = 1; 
b = 2;
print(a);
print(b+a);