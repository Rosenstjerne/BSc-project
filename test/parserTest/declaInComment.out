function null main() {
	var int a;
	
	a = 1;
	b = 2;
	print(a);
	print((b + a));
	
}
